workout.py

```diff
@@ -92,8 +92,8 @@ def countdown(exc_time):
 
 
 def run_exercise(exercise):
-    for side in exercise.sides:
-        for i in range(exercise.repetition):
+    for i in range(exercise.repetition):
```

Comment:
I'm assuming repetition means sets?  i.e. 3 sets of 10 reps on both sides?

Reply:
yeah exactly